chall04/ksappi.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+
+# Version: python3.7
+
+import sys
+import re
+import hashlib
+
+def calculateHashOfArray(array):
+	arrayStr = ''
+	for row in array:
+		arrayStr += row
+	return hashlib.md5(arrayStr.encode('utf-8')).hexdigest()
+
+try:
+	size = int(input())
+except:
+	print('Give the size of the board as an integer')
+	sys.exit()
+board = []
+if size == 0:
+	sys.exit()
+for i in range(size):
+	board.append(input())
```

Comment:
The pythonic way to write
```python
board = []
for i in range(size):
	board.append(input())
```
is
```python
board = [list(input()) for _ in range(size)]
```

Plus, you are not using `i` so I would recommend to move it a to a blank identifier.